GVRf/Framework/framework/src/main/res/raw/vcolor_vertex.vsh

```diff
@@ -1,31 +1,113 @@
+#e
```

Comment:
is this "#e" a typo?

Reply:
This PR uses a different V & F shaders than I used in PLYdemo (https://github.com/gearvrf/GearVRf-Demos/pull/647).  PLYdemo uses the same V & F shaders used by line drawing.

Reply:
I removed the "#e" and it seemed to make no difference.  I still didn't see the PLY object.
I even re-built Framework and sync'd gradle again, and same results.